manifests/mount.pp

```diff
@@ -19,9 +19,11 @@
     "types::mount::${name}::ensure is invalid and does not match the regex.")
   validate_absolute_path($name)
 
-  # ensure target exists
-  include common
-  common::mkdir_p { $name: }
+  unless $ensure == 'absent' {
```

Comment:
I am in agreement with one of the Puppet core developers that `unless` should not have ever been introduced into the language, since it is confusing. Could you please change these to "if not equal to"?